charts/garden-shoot-trust-configurator/charts/application/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,13 @@
+# SPDX-FileCopyrightText: SAP SE or an SAP affiliate company and Gardener contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: v1
+kind: ServiceAccount
```

Comment:
Note: The ServiceAccount doesn't get created automatically. 
If we skip creating the SA here, then trying to get a token for the `garden-shoot-trust-configurator` fails:

> error: failed to create token: serviceaccounts "garden-shoot-trust-configurator" not found